lib/Event/EventOptionsTrait.php

```diff
@@ -19,14 +21,14 @@
     /**
      * This array is used as key value storage for the options.
      *
-     * @var array
+     * @var OptionsResolver
```

Comment:
Yeah, your right. In this case I would suggest to change the variable to "resolver" and remove the default value. "options" is just a misleading variable name.